fecgov/fec-cms
# Occupation/Employer filter help-text for All Receipts and Indiv. Contributions

Changes to occupation/employer filters on [Individual contributions](https://www.fec.gov/data/receipts/individual-contributions/?two_year_transaction_period=2018&min_date=01%2F01%2F2017&max_date=06%2F03%2F2018) and [All receipts](https://www.fec.gov/data/receipts/) datatables.
- Rearrange `Employer` and `Occupation` filters (making Occupation first) 
- remove `hr` dividing line between the two filters' text fields
- Add help-text to  Occupation filter 

**Addresses** https://github.com/fecgov/fec-cms/issues/1976

**[How to test &#187;](#how-to-test)** 

## Impacted areas of the application:
new file:   data/templates/partials/filters/occupation-employer.jinja
modified:   data/templates/partials/individual-contributions-filter.jinja
modified:   data/templates/partials/receipts-filter.jinja

## Screenshot:
![occupation-filter](https://user-images.githubusercontent.com/5572856/40898409-19bdf476-678f-11e8-9fcf-13df5fbf6c48.jpg)
 <a id="how-to-test"></a> 

### How to test:
- Checkout an run this branch locally: `feature/occupation-filter-help-text`
- Make sure occupation and employer filters match visually to screenshot above
- Test occupation and employer filters. Tip: You can click on the details panel for an entry (the arrow in last data column) to get valid occupation and employer values to enter into the fields.
